### PR TITLE
docs: [ES] advanced tutorials, migration and recipes added.

### PR DESCRIPTION
**Pull request by Classy-Bear**
## Status
**READY**

## Breaking Changes
NO

## File changed in this PR
- [ ] README
- [ ] Cover page
- [x] Sidebar
- Introduction
  - [ ] Getting Started
  - [ ] Why Bloc?
- Core Concepts
  - [ ] bloc
  - [ ] flutter_bloc
- [ ] Architecture
- [ ] Testing
- [ ] Naming Conventions
- [ ] FAQs
- [x] Migration
- Tutorials
  - Flutter
    - [ ] Counter
    - [ ] Timer
    - [ ] Infinite List
    - [ ] Login
    - [x] Weather
    - [x] Todos
    - [x] Firebase Login
    - [x] Firestore Todos
  - AngularDart
    - [ ] Counter
  - Flutter + AngularDart
    - [ ] Github Search
- Recipes
  - Flutter
    - [x] Show SnackBar
    - [x] Navigation
    - [x] Bloc Access
- Tools
  - Extensions
    - [x] IntelliJ
    - [ ] VSCode

### CODE REVIEW

**Comment by Classy-Bear** [comment]
For [ES] documentation there are only two sections left to translate:
- AngularDart
    - Counter
- Flutter + AngularDart
    - Github Search

I will be translating it in a few weeks. When the documentation has the latest changes (v6.0.0) I will update the ES docs with these new changes, just let me know when the docs are ready.

File: docs/es/flutterlogintutorial.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Flutter Login Tutorial
+#Tutorial de iniciar sesión en Flutter
```
Comment (felangel) [inline docs/es/flutterlogintutorial.md:1]:
```suggestion
# Tutorial de iniciar sesión en Flutter
```

**Comment by felangel** [comment]
@Classy-Bear thanks for much for the amazing contributions! I really appreciate it 🙏 💯 💙 
I'll let you know once all the tutorials have been updated for v6.0.0 (so far it's just flutter_login and flutter_counter) but I'm trying to get them done as soon as possible.

**Comment by Classy-Bear** [comment]
Happy to help :). Could you please update the `CONTRIBUTING.md` file? I used the old model that doesn't has the "Type of changes". Besides that thanks for your awesome package :p

**Pull request merged**